### DISCUSSION TITLE
What tools (software & hardware) have been used by agencies to automate data collection for WZDx data feeds?

### DISCUSSION THREAD

**Discussion by CfeliceUDOT** [usdot-jpo-ode/jpo-wzdx#113] [Q&A]
The Utah Department of Transportation (UDOT) is in the process of creating a WZDx data feed. In the past, UDOT has created a mobile application that allows contractors and UDOT personnel to update information on construction projects that are affecting the traveling public. This information includes work zone locations, lane closures, road closures, and the date and time of these closures. Getting users of this mobile application to consistently enter information about work zones and closures has been very difficult. UDOT is investigating technologies to automate the collection of work zone information so that the WZDx data feed will always be current. 

Can anyone give their insight on how their agency has automated the collection of work zone information for inclusion into their WZDx data feeds? Specifically, what software or hardware (e.g. smart cones) tools were used to automate the collection of this data? Also, what problems were encountered when implementing this automation?

**Comment by sknick-iastate** [comment]
In Iowa we have been testing using smart arrow boards and connected pins/panels.  More information can be found in the [Iowa DOT work zone reference library](https://iowadot.gov/workzonereferencelibrary) but the deployment plan is available here: https://iowadot.gov/workzonereferencelibrary/docs/Smart-Arrow-Board-Deployment-Plan.pdf
